packages/cli/create-strapi-app/src/cloud.ts

```diff
@@ -0,0 +1,108 @@
+import inquirer from 'inquirer';
+import { resolve } from 'node:path';
+import { cli as cloudCli, services as cloudServices } from '@strapi/cloud-cli';
+import parseToChalk from './utils/parse-to-chalk';
+
+interface CloudError {
+  response: {
+    status: number;
+    data: string | object;
+  };
+}
+
+function assertCloudError(e: unknown): asserts e is CloudError {
+  if ((e as CloudError).response === undefined) {
+    throw Error('Expected CloudError');
+  }
+}
+
+export async function handleCloudProject(projectName: string): Promise<void> {
+  const logger = cloudServices.createLogger({
+    silent: false,
+    debug: process.argv.includes('--debug'),
+    timestamp: false,
+  });
+  let cloudApiService = await cloudServices.cloudApiFactory();
+  const defaultErrorMessage =
+    'An error occurred while trying to interact with Strapi Cloud. Use strapi deploy command once the project is generated.';
+
+  try {
+    const { data: config } = await cloudApiService.config();
+    logger.log(parseToChalk(config.projectCreation.introText));
+  } catch (e: unknown) {
+    logger.debug(e);
+    logger.error(
+      'An error occurred while trying to interact with Strapi Cloud. Use strapi deploy command once the project is generated.'
```

Comment:
Shouldn't this be saying something like "We were unable to retrieve the project creation text" rather than showing the defaultErrorMessage? Especially since it then continues and could potentially show the same message again later.

Reply:
I think we missed a return.

Should display defaultErrorMessage + return (because if we can't reach the config from API, its useless to continue the process.